packages/server/src/routes/index.js

```diff
@@ -25,23 +27,28 @@ export default app => {
   app.put(    '/users/:address/notifications',        asyncMiddleware(users.notifications.change))
 
 
-  // other endpoints
-  app.post('/login', asyncMiddleware(admins.login))
+  /*********** Old routes ***********/
```

Comment:
Should we just remove these old routes? And do we want to consider `/reveals` an old route?

Reply:
Well, we are currently using them. I mean "old" in terms of routes we haven't sanitized yet with the RESTful rules we applied to the new ones, would love to update these in the future too

Reply:
Left an issue to tackle that here https://github.com/aragonone/court-backend/issues/135 :)